cpusupport/cpusupport.h

```diff
@@ -94,6 +94,26 @@
 	int								\
 	cpusupport_ ## arch ## _ ## feature ## _detect_1(void)
 
+/**
+ * CPUSUPPORT_VALIDATE(hwvar, success_value, cpusupport_checks, check):
+ * Check if we can enable ${success_value}, given the ${cpusupport_checks} and
+ * ${check}; if so, write to ${hwvar}.  If the ${cpusupport_checks} pass but
+ * the ${check} is non-zero, produce a warning which includes a stringified
+ * ${success_value}, then fallthrough.
+ */
+#define CPUSUPPORT_VALIDATE(hwvar, success_value, cpusupport_checks,	\
+    check) do {								\
```

Comment:
Is the misaligned `\` a github display issue?

Reply:
Yes.  :(